doc/template.txt

```diff
@@ -108,9 +108,9 @@ plug-in:
 		useful on a Windows box where `*` is not allowed in file
 		names. The default configuration converts underscores (`_`)
 		into regular expression wildcards (`.*`).
-		
```

Comment:
Ah, stray end-of-line whitespace... Nice catch.

It could have gone in a separate PR, but let's just merge this one as-is :wink: